lib/validate.ts

```diff
@@ -12,47 +12,50 @@ import { componentsDefinitionSchema_v1_2_x } from './components-schema-v1_2_x';
 import { componentsDefinitionSchema_v1_3_x } from './components-schema-v1_3_x';
 import { componentsDefinitionSchema_v1_4_x } from './components-schema-v1_4_x';
 import { componentsDefinitionSchema_v1_5_x } from './components-schema-v1_5_x';
+import { componentsDefinitionSchema_v1_6_x } from './components-schema-v1_6_x';
 
 import { parseDefinition } from './parser';
 import {
```

Comment:
Did you trigger your import sorting code :)?

Reply:
I used "Optimize imports" because IntelliJ indicated some imports could be taken from barrel files and that sort too :-).